### ISSUE TITLE
Undefined while saving page as template but not working in individual section

### ISSUE THREAD

**Issue by tareqhassan** [elementor/elementor#7704]
![Screenshot_2](https://user-images.githubusercontent.com/9674371/55389620-7120c700-5557-11e9-97b6-84a07be47c76.jpg)

I've Elementor 2.5.11 & Elementor Pro 2.5.3 when I try to save the entire page as template??

Already tried the following attempts -
1. Disable all plugins and revert to new theme
2. PHP version 7.3.3
3. Memory limit: 768M
4. Time limit 500

The issue is only occured when I'm trying to save the entire page as a page template. But individula section save smoothly. Not only that, if I edit older pages, there are no issues. Only happen for newly created pages. 

At the log section, I've this errors - 

JS: showing 4 of 4
JS: 2019-01-22 04:38:13 [error X 10][https://demo.domain.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.3:2:167360] Cannot read property \'errors\' of undefined 
JS: 2019-02-15 22:27:55 [error X 3][https://demo.domain.com/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.4.4:2:15986] Cannot read property \'model\' of undefined 
JS: 2019-03-29 23:58:58 [error X 7][https://demo.domain.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.10:2:53459] Cannot read property \'noUiSlider\' of undefined 
JS: 2019-04-01 20:51:39 [error X 2][https://demo.domain.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.11:2:83388] Cannot set property \'controls\' of undefined 

## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
This is a specific issue and not a global issue.

Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed by me and reference this issue.

thanks.

**Comment by tareqhassan** [comment]
Thanks for the reference. I've found another issue here which looks like the main cause. 

Ref: https://github.com/elementor/elementor/issues/7701

When the page elements are smaller, there are no issues, but when it is large, it causes the problem. 

However, I'm emailing to the support team for further investigations.

**Comment by shilo-ey** [comment]
@tareqhassan 

Thanks! we will take a look in the other issue

**Comment by Oloyede1** [comment]
![ELEMENTOR](https://user-images.githubusercontent.com/65760960/82651036-83563c80-9c13-11ea-9c90-224d1dc41096.png)

Hi,

I have the same issue when I try to save the entire page as template and i have tried many options like Disable all plugins and revert to new theme, PHP version 7.3.3. Memory limit: 2000M. Time limit 180. Please i will appreciate any useful hint.

Many thanks.

**Comment by Pablo-Aldana** [comment]
Exact same problem here. is there a solution for this other than contact the support? I just have a vanilla WP installation with Element and I get this issue with all pages

**Comment by freephotosdepot** [comment]
Ok so I am also having the same problem when i try to load a single product template. i click connect to cloud, it acts like its loading and then the error pops up. i have already had hosting check stuff and i already disabled all the plugins. i am running on oceanwp! I have already emailed support as well. Waiting on a response!

**Comment by raddnoa** [comment]
[Solved the elementor undefined error when saving template](https://github.com/elementor/elementor/issues/7701#issuecomment-717336634)
